artemis-rest/src/main/java/org/apache/activemq/artemis/rest/topic/SubscriptionsResource.java

```diff
@@ -376,7 +376,7 @@ private QueueConsumer recreateTopicConsumer(String subscriptionId, boolean autoA
       if (subscriptionExists(subscriptionId)) {
          QueueConsumer tmp = null;
          try {
-            tmp = createConsumer(true, autoAck, subscriptionId, null, consumerTimeoutSeconds * 1000, false);
```

Comment:
I thought the compiler would solve this by converting it to long. The outcome is a long.